scripts/genomes/parse_genome_assemblies.py

```diff
@@ -0,0 +1,208 @@
+"""Parse genome assemblies from NCBI by their level of completion.
+
+This script fetches current and historical Assembly Reports from NCBI, and
```

Comment:
Some context about what historical Assembly Reports are and why they should or should not be included would be helpful

Reply:
This context is provided in the [`get_assembly_report`](https://github.com/broadinstitute/single_cell_portal/pull/37/files#diff-590e34483c0ec608edef9d946bbe8e15R50) docstring.  I've noted that in this module docstring in my current topic branch.